src/workerd/server/tests/python/BUILD.bazel

```
@@ -21,6 +21,8 @@ py_wd_test("env-param")
 py_wd_test(
     "asgi",
     skip_python_flags = ["0.27.1"],
+    # TODO: investigate failure!
+    snapshot_variant = False,
```

Just to make it a bit clearer:

```suggestion
    snapshot_generating_variant = False,
```